avm/res/insights/data-collection-rule/main.bicep

```
@@ -152,15 +152,19 @@ output resourceId string = dataCollectionRuleProperties.kind == 'All' ? dataColl
 output resourceGroupName string = resourceGroup().name
 
 @description('The location the resource was deployed into.')
-output location string = dataCollectionRuleProperties.kind == 'All'
-  ? dataCollectionRuleAll.location
-  : dataCollectionRule.location
+output location string = dataCollectionRuleProperties.kind == 'All' ? dataCollectionRuleAll.location : dataCollectionRule.location
```

```suggestion
output location string = dataCollectionRuleProperties.kind == 'All' ? dataCollectionRuleAll!.location : dataCollectionRule!.location
```
To suppress warnings